app/src/main/java/org/ometa/lovemonster/service/LoveMonsterClient.java

```diff
@@ -340,6 +346,31 @@ void onAuthenticationFailure() {
         });
     }
 
+    public void getUserFromUsername(@NonNull String username, @NonNull final UserLookupResponseHandler responseHandler) {
+        final URIBuilder url = buildUrl("/api/v1/users/" + username);
```

Comment:
You could gsub/scrub the input to prevent that, but because this is part of the path there isn't any way I know of besides concatenation.